gp-includes/template.php

```diff
@@ -98,7 +98,7 @@ function gp_nav_menu( $location = 'main' ) {
 	$items = gp_nav_menu_items( $location );
 
 	// Get the current URI.
-	$current_uri = str_replace( home_url(), '', gp_url_current() );
+	$current_uri = str_replace( home_url(), '', trailingslashit( gp_url_current() ) );
```

Comment:
It is a bit weird when you put a URL without the ending slash and the current link in the menu is marked. What do you think about adding a redirect to the URL with the ending slash when the user goes to a URL without this ending slash?

```suggestion
	$current_uri           = str_replace( home_url(), '', trailingslashit( gp_url_current() ) );
	$uri_no_trailing_slash = str_replace( home_url(), '', gp_url_current() );
	if ( $current_uri !== $uri_no_trailing_slash ) {
		wp_redirect( $current_uri );
	}
```

Reply:
I like the idea, but isn't redirecting only on `gp_nav_menu()` for matching menus with parent items CSS too late?
What if a specific page don't load `gp_nav_menu()`?
I think that the redirect should be immediatly on page load, while checking the current URL agains the known routes.